modules/shortcodes/instagram.php

```diff
@@ -190,10 +191,8 @@ function jetpack_instagram_handler( $matches, $atts, $url ) {
 			false,
 			true
 		);
-		// there's a script in the response, which we strip on purpose since it's added by this ^ script
-		$ig_embed = preg_replace( '@<(script)[^>]*?>.*?</\\1>@si', '', $response_body->html );
-
-		return $ig_embed;
+		//no need for preg replace as the js file is automatically omitted via api call
```

Comment:
I think you can remove this comment as well now; folks won't know why the comment mentions `preg_replace` when there is no `preg_replace` in there.

```suggestion

```

Reply:
comment has been removed, and code [committed](https://github.com/Automattic/jetpack/pull/11920/commits/cbb2f055c95661a9541c3c8f6b6f8d9a973f2932)